L/LibPQ/build_tarballs.jl

```diff
@@ -34,8 +34,13 @@ if [[ "${target}" == i686-linux-musl ]]; then
     # other tests.
     sed -i 's/cross_compiling=no/cross_compiling=yes/' configure
 fi
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-includes=$prefix/include --with-libraries=$prefix/lib --without-readline --without-zlib --with-openssl
+if [[ "${target}" == *"linux"* ]] ; then
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-includes=$prefix/include --with-libraries=$prefix/lib --without-readline --without-zlib --with-openssl --with-gssapi
+ else
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-includes=$prefix/include --with-libraries=$prefix/lib --without-readline --without-zlib --with-openssl
+ fi
```

Comment:
I don't like long duplicate lists of flags.
```suggestion
FLAGS=()
if [[ "${target}" == *-linux-* ]]; then
    FLAGS+=(--with-gssapi)
fi
./configure --prefix=${prefix} \
    --build=${MACHTYPE} \
    --host=${target} \
    --with-includes=${includedir} \
    --with-libraries=${libdir} \
    --without-readline \
    --without-zlib \
    --with-openssl \
    "${FLAGS[@]}"
```

Reply:
I pushed these changes, together with the fix for FreeBSD

Reply:
Thanks a lot.

Great you found out what was the problem for FreeBSD.